app/questionnaire/rules.py

```diff
@@ -23,6 +23,8 @@ def evaluate_rule(when, answer_value):
         return True
     elif condition == 'contains' and isinstance(answer_to_test, list) and match_value in answer_to_test:
         return True
+    elif condition == 'not contains' and isinstance(answer_to_test, list) and match_value not in answer_to_test:
```

Comment:
Like this one? https://github.com/ONSdigital/eq-survey-runner/pull/902/files#diff-c0704fcf066dc43ec2f73117f8ca828eR110